makers/senior_engineer.md

```diff
@@ -1,35 +1,78 @@
-
-* Successfully performed the role of an Engineer at Mariana Tek or equivalent role elsewhere
-* Performed work that was evaluated to be done at ‘high quality’ by peers
-* Succeeded as an engineer when presented with projects of increasing complexity over time across multiple areas or domains of the code/product
-* Demonstrated ability to learn from mistakes
-* Demonstrated ability to unblock themselves and other team members  
-* Earned a high level of trust amongst team to own a task/project without needing much supervision
-* Shown they take responsibility for their code: follow through with it and address any issues that result from it in the future
-* A high level of understanding about where their work fits in and how to be useful and successful on their team
-* A high level of craftsmanship about their work, the end experience of the user, and impact on overall product
-* Demonstrated good communication skills with other engineers and teammates, including being productive with their emotions; proposes solutions when articulating problems
-
-### Is expected to
-
-* Become an expert with multiple areas or domains of the code/product
-* Show curiosity to not only learn new things but fully understand how they work 
-* Be highly productive - have a reputation for getting things done quickly and efficiently
-* Be a mentor for other engineers
-* Require very little oversight from manager/team leads
-* Own and complete full projects beginning with identifying and communicating the problems to be solved, getting and incorporating feedback on proposed architectural solutions, and making a final decision as the owner of a project.
-* Deconstruct a problem into an executable action plan for themselves and other engineers - also perform them in a high quality way without issue
-* Consistently tie back a project to a team goal
-* Set and maintain high individual and team expectations
-* Actively participate in frequent code/design/architecture reviews
-* Be able to communicate well with all engineers regardless of seniority
-* Generate support for a company/team decision
-* Work across engineering teams (dev, QA, systems) to ensure new code launches smoothly and customer needs are met
-* Monitor the outcomes of team’s actions to learn from mistakes and replicate successes
-* Interview prospective candidates and provide thoughtful evaluations and feedback
+<!-- TODO: Add summary -->
+
+## Qualifications
+
+- Successfully performed the role of Engineer at Mariana Tek or equivalent role elsewhere.
+- Succeeded as an engineer when assigned projects of increasing complexity over time across multiple product domains.
+- Consistent track record of writing high-quality code with very little oversight beyond high-level direction.
+- Demonstrated ability to learn from mistakes.
+- Demonstrated ability to unblock themselves and others.
+- Communicates effectively with colleagues at all levels.
+## Responsibilities & Expectations
+
+<!-- TODO: Add summary -->
```

Comment:
I think there's something interesting happening here - as I am progressing through your proposed changes, I'm starting to question what would value a summary section at the top of the `Responsibilities & Expectations` section would have given the clarity and thoroughness of the content below within the sub-sections `Technical Skills`, `Business Knowledge`, `Communication`, et cetera?

My vote here is that we forego a summary section in the interest of:

- Getting to merge with fewer blocks
- Keeping the content actionable and clear
- Leaning on the clarity of the surrounding text

TLDR - I simply don't think there's a huge amount of value in the summary compared to the vast value presented already in the sub-sections.

Reply:
My general idea for the summary section would be a sentence or two outlining the high-level function/goal of each role, more articulately than what I've noted below:

- Engineer I: A new engineer focused on learning the ropes of software development, learning to write code consistent with team standards, growing their development skills
- Engineer II: A solid contributor, often self-sufficient & becoming more so, developing expertise in a particular domain, contributing more to the technical design process with guidance from more senior engineers
- Senior Engineer: A strong independent contributor, developing your tech leadership skills, handling end-to-end responsibility for projects of increasing complexity
- Staff Engineer: A team leader? A cross-team leader? Provide high-level technical guidance across your team / teams, help colleagues build and produce better outcomes
- Principal Engineer: Primarily focusing on long-term architectural and technology needs for parts of the business
- CTO: An organizational leader, responsible for setting company-wide technical direction

Reply:
🤔 Okay - this is growing on me! I trust your gut here - can go either way of course, I do like your proposals and think we can get good value of these summarizations 👍